src/ralph/backends/database/es.py

```diff
@@ -41,16 +69,30 @@ def get(self, chunk_size=500):
                 bytes(json.dumps(document.get("_source")) + "\n", encoding="utf-8")
             )
 
-    def to_documents(self, stream, get_id):
+    def to_documents(self, stream, get_id, op_type):
```

Comment:
```suggestion
    def to_documents(self, stream, get_id):
```
Maybe we could use `self.opt_type` ?

Reply:
You are right as we already depend on the class instance `index` attribute.